doc/machine-auto.rst

```diff
@@ -6,7 +6,7 @@ machine:
         | type: ``str``
         | argument path: ``machine/batch_type``
 
-        The batch job system type. Option: Slurm, PBS, LSF, Shell, DpCloudServer
+        The batch job system type. Option: Slurm, PBS (for OpenPBS only), Torque (for Torque version of pbs), LSF, Shell, DpCloudServer
```

Comment:
The file is automatically generated from `machine.py`. Please modify there instead.
https://github.com/deepmodeling/dpdispatcher/blob/376137101b49a105402bfdd577b645f43aee3d21/dpdispatcher/machine.py#L287

Reply:
OK. Please check #122.